coalib/bearlib/abstractions/Linter.py

```diff
@@ -207,6 +207,9 @@ def check_prerequisites(cls):
             :return:
                 True if operational, otherwise a string containing more info.
             """
+            if not super(LinterBase, cls).check_prerequisites():
+                return False
+
```

Comment:
@arush0311 Oh! So it was meant serious :) Sorry! The problem with just returning the result of the `super().check_prerequisites()` call is that in the `True` case any code afterwards won't be evaluated anymore